### PR TITLE
refactor: use our base image to build UI's image

### LINKED ISSUE #23: Use LH node base image
Currently we are using a node vanilla base image in our docker UT image https://github.com/littlehorse-enterprises/lh-user-tasks/blob/main/Dockerfile#L1

To enable TLS we should use nginx, we have create a base image for that https://github.com/littlehorse-enterprises/alpine-nginx-nodejs

nginx configuration here https://github.com/littlehorse-enterprises/alpine-nginx-nodejs/blob/main/nginx.conf

Goal
Adapt our UT image to use the LH base image

### PR DESCRIPTION
**Pull request by HazimAr**
closes #23 
I tested the standalone locally and everything works.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
-ENV NODE_ENV=production \
-    PORT=3000 \
-    HOSTNAME="0.0.0.0"
 EXPOSE 3000
+ENV PORT=3000
+ENV HOSTNAME="0.0.0.0"
+ENV NODE_ENV="production"
 
-ENTRYPOINT [ "./entrypoint.sh" ]
+ENTRYPOINT [ "./entrypoint.sh" ]
```
Comment (mijailr) [inline Dockerfile:17]: This overwrites the base image entrypoint

Comment (HazimAr) [reply Dockerfile:17]: image
It does not overwrite it as its in the `/app` directory

**Comment by HazimAr** [comment]
@mijailr feel free to merge the PR once SSL has been tested

**Review by mijailr — APPROVED** [review]
Works fine on my local. Would be nice if you add how to run with SSL in the readme.

This is the command that I've used. Please modify to reflect a real usable scenario on the readme.

```shell
docker run --rm -it \
    -e SSL=enabled \
    -v ./ssl:/ssl \
    -e NEXTAUTH_URL='http://localhost:3000' \
    -e NEXTAUTH_SECRET='<any secret here>' \
    -e KEYCLOAK_HOST='http://localhost:8888' \
    -e KEYCLOAK_REALM='default' \
    -e KEYCLOAK_CLIENT_SECRET=' ' \
    -e LHUT_API_URL='http://localhost:8089' \
    -e KEYCLOAK_CLIENT_ID='123' \
    -p 3000:3000 -p 3443:3443 demo
```

**Comment by mijailr** [comment]
Also this command to generate the local certificate.

```shell
openssl req -x509 -out tls.crt -keyout tls.key \
  -newkey rsa:2048 -nodes -sha256 \
  -subj '/CN=localhost' -extensions EXT -config <( \
   printf "[dn]\nCN=localhost\n[req]\ndistinguished_name = dn\n[EXT]\nsubjectAltName=DNS:localhost\nkeyUsage=digitalSignature\nextendedKeyUsage=serverAuth")
```

**Comment by HazimAr** [comment]
@sauljabin I will leave the rest of this PR to you for testing / QA, I have added a script and updated the `README` to accommodate @mijailr's request.

@sauljabin feel free to merge the PR when you are done with it.

**Pull request merged**